extension/chrome/elements/compose-modules/compose-send-btn-module.ts

```diff
@@ -249,9 +249,9 @@ export class ComposeSendBtnModule extends ViewModule<ComposeView> {
   private doSendMsgs = async (msgObj: MultipleMessages): Promise<SendMsgsResult> => {
     const sentIds: string[] = [];
     const supplementaryOperations: Promise<void>[] = [];
-    const supplementaryOperationsErrors: any[] = []; // tslint:disable-line:no-unsafe-any
+    const supplementaryOperationsErrors: unknown[] = []; // tslint:disable-line:no-unsafe-any
```

Comment:
you may remove `// tslint:disable-line:no-unsafe-any` comment if you already converted `any` to `unknown`
There are several places which contains this comment

Reply:
oh - ok, thanks... I'll be removing them as well. thank you!